### PR TITLE
PairV2 entity: add unit testing, clean up, and fix number overflow issue

### PR DESCRIPTION
**Pull request by gaepsuni**
This PR adds some tests for the new `PairV2` , `TradeV2`, and `RouteV2` entities.

Also, some unused functions are cleaned up, and `addLiquidityParameters()` return types for amounts are changed to `string` to avoid number overflow

### CODE REVIEW

File: src/v2entities/pair.ts [hunk 1]
```diff
@@ -329,20 +278,36 @@ export class PairV2 {
...
+    const amountXMin = new Fraction(ONE)
+      .add(amountSlippage)
+      .invert()
+      .multiply(_amountX)
+      .quotient.toString()
+    const amountYMin = new Fraction(ONE)
+      .add(amountSlippage)
+      .invert()
+      .multiply(_amountY)
+      .quotient.toString()
```
Comment (gaepsuni) [inline src/v2entities/pair.ts:310]: Changed type to `string` to avoid overflow

**Review by 0xmurloc — APPROVED** [review]
LGTM
